nix/drun.nix

```diff
@@ -17,6 +17,8 @@ pkgs:
 
       cargoSha256 = "sha256-dhDXhVNAAHzLRHdA6MpIGuoY76UhiF4ObeLO4gG/wo4=";
 
+      patches = [./now.patch];
```

Comment:
Andriy Berestovskyy's heroic patch!

This can be removed when we import the fixed `drun` from GH...